util/buildRelease/test_install.bash

```diff
@@ -55,17 +56,48 @@ then
       EXITSTATUS=$TEST_STATUS
     fi
   fi
+
+  # Check that bin/chpldoc was created
+  if [ ! -f "$myprefix/bin/chpldoc" ]
+  then
+    echo "--prefix install: $myprefix/bin/chpldoc was not installed!" 1<&2
+    EXITSTATUS=1
+  else
+    # Try running chpldoc on a test program
+    (
+      export PATH=$myprefix/bin:$PATH
+      rm -Rf "$mydocstmp"
+      mkdir -p "$mydocstmp"
+      cd "$mydocstmp"
+      chpldoc $CHPL_CHECK_HOME/examples/primers/chpldoc.doc.chpl
+      exit $?
+    )
+    TEST_STATUS=$?
+    if [ $TEST_STATUS -ne 0 ]
+    then
+      echo "--prefix install: running chpldoc failed" 1>&2
+      EXITSTATUS=$TEST_STATUS
+    fi
+  fi
+
 fi
 
 if [ $EXITSTATUS -eq 0 ]
 then
   # Next, check installation to a chpl-home
-  ./configure --chpl-home="$myhome" && make && make install
+  ./configure --chpl-home="$myhome" && make && make mason && make chpldoc && make install
```

Comment:
Was including mason in this PR intentionally?  I think it's good to add, but it should be called out in your PR/merge message if it was intentional

Reply:
Yes, it was intentional. It actually was already added to the build in the earlier part of this script. I added it here redundantly for clarity. It should not change behavior.

Reply:
Well... actually it was probably necessary to get the mason checking in this configuration to work due to the `rm` of `bin` elsewhere in the script.